assignments/objects.js

```diff
@@ -19,6 +19,54 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  "id" : 1,
+  "name" : "Mitzi",
+  "email" : "mmelloy0@psu.edu",
+  "gender" : "F"
+}
+const intern2 = {
+  "id" : 2,
+  "name" : "Kennan",
+  "email" : "kdiben1@tinypic.com",
+  "gender" : "M",
+   speak : function() {
+  // console.log("Hello my name is Kennan!");
+  console.log(intern2.speak()
```

Comment:
Here is where you're writing the speak method on the intern2 object, so you you should have the commented code be uncommented and line 35 should just be removed (it's lacking a parenthesis at the end as well).